subworkflows/local/short_variant_calling/main.nf

```diff
@@ -70,18 +71,37 @@ workflow SHORT_VARIANT_CALLING {
 
     // Multisample
     GLNEXUS( ch_glnexus_in, ch_bed )
-    TABIX_GLNEXUS(GLNEXUS.out.bcf)
+
+    // Add allele count tag to multisample bcf
+    BCFTOOLS_FILLTAGS ( GLNEXUS.out.bcf )
+    ch_versions = ch_versions.mix(BCFTOOLS_FILLTAGS.out.versions)
+
+    // Decompose and normalize variants
+    BCFTOOLS_FILLTAGS.out.vcf
+        .concat( BCFTOOLS_SORT_DV_VCF.out.vcf)
+        .map { meta, vcf -> [ meta, vcf, [] ] }
+        .set { bcftools_norm_in }
+
+    BCFTOOLS_NORM ( bcftools_norm_in, ch_fasta )
+    ch_versions = ch_versions.mix(BCFTOOLS_NORM.out.versions)
```

Comment:
same here

Reply:
Like above, I see that most if not all nf-core subworkflows also do it like that (e.g. https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/fastq_align_bwa/main.nf).